File: python-service/microanalyzer/models/object_.py
class Object_:
    def __init__(self, isInterface: bool, isAnnotation: bool, simpleName: str, fullName: str, filePath: str,
                 serviceName: str, content: str):
        self.isInterface = isInterface
        self.isAnnotation = isAnnotation
        self.simpleName = simpleName
        self.fullName = fullName
        self.filePath = filePath
        self.serviceName = serviceName
        self.content = content

File: ruby-service/app/models/object_.rb
# frozen_string_literal: true

class Object_
  attr_accessor :simpleName, :fullName, :filePath, :serviceName, :content, :isInterface, :isAnnotation
  def initialize(simpleName, fullName, filePath, serviceName, content)
    @simpleName = simpleName
    @fullName = fullName
    @filePath = filePath
    @serviceName = serviceName
    @content = content
    @isInterface = false
    @isAnnotation = false
  end

  def to_hash
    {
      'simpleName' => @simpleName,
      'fullName' => @fullName,
      'filePath' => @filePath,
      'serviceName' => @serviceName,
      'content' => @content,
      'isInterface' => @isInterface,
      'isAnnotation' => @isAnnotation
    }
  end
end
